### PR TITLE
Container structure test

### PR DESCRIPTION
**Pull request by WengerK**
### 💬 Describe the pull request

Add testing to ensure container structure using Google Container structure test framework

### 🔢 To Review
Steps to review the PR:
1. Use the local `./test.sh` script on your machine

```
./test.sh --test=7.3
./test.sh --test=7.3-node12
```

2. Github Actions should also run every tests

### CODE REVIEW

File: php/7.3/tests/config--7.3-node12.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+schemaVersion: '2.0.0'
+commandTests:
+  - name: "PHP Version"
+    command: "php"
+    args: ["-v"]
+    expectedOutput: ["PHP 7.3.24"]
```
Comment (WengerK) [inline php/7.3/tests/config--7.3-node12.yml:6]:
```suggestion
    expectedOutput: ["^PHP 7.3.\d+"]
```

File: php/7.1/tests/config--7.1-node10.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+schemaVersion: '2.0.0'
+commandTests:
+  - name: "PHP Version"
+    command: "php"
+    args: ["-v"]
+    expectedOutput: ['^PHP 7\.1\.\d+']
+  - name: "Node Version"
+    command: "node"
+    args: ["-v"]
+    expectedOutput: ['^v10\.\d+\.\d+']
```
Comment (gido) [inline php/7.1/tests/config--7.1-node10.yaml:10]: This looks very repetitive. Is there an alternative ?

Comment (WengerK) [reply php/7.1/tests/config--7.1-node10.yaml:10]: I don't found any - I could use a "base file" that use/load the [node-version]-config.yaml file

Comment (yannlugrin) [reply php/7.1/tests/config--7.1-node10.yaml:10]: Maybe it's possible to use ENV there? as so use the matrix information? if we have them.

Comment (WengerK) [reply php/7.1/tests/config--7.1-node10.yaml:10]: I will merge and open an new Issue explaining our need to simplify this code by using ENV variables if possible

**Pull request merged**